spec/requests/reactions_spec.rb

```diff
@@ -211,20 +211,41 @@
       before do
         user.setting.update_column(:experience_level, 8)
         sign_in user
+      end
+
+      it "creates reaction" do
         post "/reactions", params: {
           reactable_id: article.id,
           reactable_type: "Article",
           category: "readinglist"
         }
-      end
 
-      it "creates reaction" do
         expect(Reaction.last.reactable_id).to eq(article.id)
       end
 
       it "creates rating vote" do
-        expect(RatingVote.last.context).to eq("readinglist_reaction")
-        expect(RatingVote.last.rating).to be(8.0)
+        post "/reactions", params: {
```

Comment:
I moved posting the request out of the `before` block because in the last spec I want to update the user before doing that.